src/router/index.ts

```diff
@@ -30,9 +20,19 @@ const router = createRouter({
       component: () => import("@/views/SuccessView.vue")
     },
     {
-      path: "/:pathMatch(.*)*",
+      path: "/404",
```

Comment:
I would not remove this, since you still can type gibberish in the url and it should land you on 404

Reply:
Better use named route

Reply:
Like if someone has multiple url fragments. Sorry that slipped my mind